### ISSUE TITLE
Usage with uMatrix while blocking all requests except CSS and images.

### ISSUE THREAD

**Issue by bash550** [Synzvato/decentraleyes#270]
Assuming you block all requests except CSS and images in uMatrix, would there still be a need for Decentraleyes, and if so, would it work properly if those rules are applied?  
As it is, it doesn't seem to work on chrome, though I am a bit of a noob.

**Comment by beerisgood** [comment]
uMatrix doesnt replace Decentraleyes. Read what Decentraleyes do.
Also read which stuff you need to allow, so it can work.

**Comment by criztovyl** [comment]
While Decentraleyes will do nothing_* in your case, it meight be useful to reduce your footprint for websites where you need to (temporary) whitelist CDNs because the sites are broken otherwise.

_* Currently not sure if Decentraleyes also replaces CSS from CDNs, if so, that files would be replaced, if Decentraleyes has them.

**Comment by Thorin-Oakenpants** [comment]
> Assuming you block all requests except CSS and images in uMatrix,

Look, I'm a kind of default block *everything* and whitelist. Your hypothetical setup is similar to mine, except I limit CSS and images to 1st party
![um](https://user-images.githubusercontent.com/16656956/40194008-8b1cdda2-59f8-11e8-9efe-623c787b1288.png)

I also only allow 1st party in uBO. And back in pre FF57 days I also used NoScript (legacy). These stats (around 21%) are since I moved to uBO web extension, prior to that the ratio was about 7% (because NoScript blocked most 3rd party requests even getting to uBO)
![ubo](https://user-images.githubusercontent.com/16656956/40194070-b7a3a55e-59f8-11e8-85d2-a3daaba42f70.png)

> Assuming you block all requests except CSS and images in uMatrix

Back to this point... as criztovyl says, you need to whitelist in order to be able to actually use chunks of the internet. So here I am, pretty much locking down *anything* imaginable and allowing thru the basics where I *really* need them (and I live in a pretty much broken internet because of it)

Here is my DC stats right now
![dc-now](https://user-images.githubusercontent.com/16656956/40194259-4fb07c6e-59f9-11e8-96a4-c8b81a92c8a0.png)

And here's what they were seven months ago (https://github.com/Synzvato/decentraleyes/issues/214#issuecomment-337953662)
![dc-old](https://user-images.githubusercontent.com/16656956/40194271-57daa98c-59f9-11e8-9b5c-665ddfbd3484.png)

That's 12,805 locally injected resources for 7 months browsing, and I run a **tight* ship ..

tl;dr: to answer you question, *hell yeah*, there is a need for Decentraleyes in *any* configuration

**Comment by bash550** [comment]
Thanks, @Thorin-Oakenpants , @criztovyl and  @beerisgood .

**Comment by criztovyl** [comment]
You're welcome.
